### PR TITLE
Tips

### PR DESCRIPTION
**Pull request by Mnemotechnician**
## About the PR
Added 41 new in-game tips, commented out some irrelevant ones that don't apply to frontier; also gave some old ones a better/generic replacement.

## Why / Balance
Why not

## Technical details
Yml change

## Media
N/A

- [X] This PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**
N/A

### CODE REVIEW

File: Resources/Prototypes/Datasets/tips.yml [hunk 1]
```diff
@@ -123,3 +123,46 @@
...
+  - "While holding a bag with 120 free space, you can force a small person inside. Be careful, they may get mad at you!"
+  - "Staying in deep space induces severe stress and often leads to a sudden sleep disorder. You can hire an emotional support crewmember to help you overcome it!"
+  - "You can use your hands to carry people! Alt-click a person while having two free hands, or use the context menu to do so."
+  - "The Vulpkanin will love you if you keep petting them."
+  - "When you enter cryosleep, your body remains stored for up to 60 minutes! As a ghost, you can use the 'un-cryo' button to wake up and return to it!"
+  - "If you accidentally left the game for a long time, you can still check on your character. Perhaps someone shoved them into a cryopod and you can use the 'un-cryo' button?"
+  - "If you see a person who's fallen SSD, make sure to drag them to cryo and shove them into a cryopod. It will prevent them from dying, and will allow the player to return to the body later!"
+  - "Vending machine food is bad for your health! Watch out for food trucks such as the Skipper or the McCargo - they can offer delicious food for a reasonable price!"
```
Comment (Cheackraze) [inline Resources/Prototypes/Datasets/tips.yml:142]: maybe include another note about how Chef cooked food has more flavor, and will give you a speed buff

**Review by Cheackraze — APPROVED** [review]

File: Resources/Prototypes/Datasets/tips.yml (hunk 1 above)
Comment (Mnemotechnician) [reply Resources/Prototypes/Datasets/tips.yml:142]: I dunno, can you give me an example?

Comment (Cheackraze) [reply Resources/Prototypes/Datasets/tips.yml:142]: "Meals cooked by chefs are more filling, more flavorful, and can even boost your movement speed for a short time"

Comment (Mnemotechnician) [reply Resources/Prototypes/Datasets/tips.yml:142]: Yea, added it

**Comment by Cheackraze** [comment]
also this is conceptually approved, but I'd kinda like to keep the PR open for a few days and gather feedback from people on discord about potentially more frontier-specific tips

File: Resources/Prototypes/Datasets/tips.yml [hunk 2]
```diff
@@ -123,3 +123,47 @@
   - "Monkeys have a rare chance to be sentient. Ook!"
   - "You can tell if an area with firelocks up is spaced by looking to see if the firelocks have lights besides them."
   - "Instead of picking it up, you can alt-click food to eat it. This also works for mice and other creatures without hands."
+  #------------#
+  #  Frontier  #
+  - "Smaller shuttles offer greater maneuverability, while bigger ones offer a wider and better spectre of services. Choose wisely!"
```
Comment (TsjipTsjip) [inline Resources/Prototypes/Datasets/tips.yml:128]: spectre -> spectrum

File: Resources/Prototypes/Datasets/tips.yml [hunk 3]
```diff
@@ -123,3 +123,47 @@
...
+  - "As a crewmemeber, you can start a mutiny if your captain refuses to pay you. Be careful though, as the captain has the permission to execute the mutiny leader!"
+  - "Your PDA has a pre-installed news application. Check it out, perhaps the local reporter posted some interesting news?"
+  - "There's a special app installed in your PDA, which allows you to see posted bounties. If you can fulfill one of them, you can ask the local SR or Sheriff for a reward!"
+  - "Remember, Frontier Outpost has a 200m wide safezone. It's forbidden to seriously harm people or property in any way within that limit!"
+  - "If you're using an IFF console, remember to turn your IFF on near frontier, or you'll risk facing security on your ship!"
+  - "Going on an expedition alone is very risky! Make sure to always have a crewmate to save you in case you get ambushed by a xeno queen."
+  - "Expeditions can yield rare and useful items, and not all of them are legal. Try not to leave those lying on the floor, as security can board you at any moment!"
+  - "Sometimes there's a Station Traffic Controller on the frontier outpost. Make sure to contact them before docking, or you may face a fine!"
```
Comment (TsjipTsjip) [inline Resources/Prototypes/Datasets/tips.yml:165]: Could be useful to refer to the traffic encryption key in the survival box. Might be worth a separate tip entirely.

File: Resources/Prototypes/Datasets/tips.yml [hunk 4]
```diff
@@ -123,3 +123,47 @@
...
+  - "As a medic, you should cooperate with others when there's a medical emergency! It never helps when two medical ships arrive at the same emergency signal."
+  - "As a medic, you can ask the local SR for a medical PDA. It can double as a health analyzer!"
+  - "Medical ships often have access to the hypospray. An experienced chemist can turn it into a lethal weapon, or use it to safely put people to sleep."
+  - "If you're hurt, try to ask for medical help on the radio. Medics are often eager to heal the wounded, as their job usually gets boring after hours of waiting."
+  - "Always carry a GPS with yourself! It can help you locate your station, or tell your location to others and get saved."
+  - "Smuggling is very profitable, but also very risky! Remember to have an IFF console and hide your IFF before going for a dead drop."
+  - "As a smuggler, try to destroy any evidence of your crimes. Remember, your PDA has a notepad app, which can be used to store important notes in a digital form."
+  - "Security has a special IFF console that completely hides their ship on the radar! If you're an outlaw, watch out for these invisible prowlers!"
```
Comment (TsjipTsjip) [inline Resources/Prototypes/Datasets/tips.yml:156]: these -> those

(These refers to a concept referred to previously, but a Prowler specifically is never mentioned before. Alternatively, refer to the ship name in the first section.)

**Review by TsjipTsjip — COMMENTED** [review]
Couple of things I noticed about the new set of tips, I do love these additions tho.

File: Resources/Prototypes/Datasets/tips.yml (hunk 4 above)
Comment (arimah) [reply Resources/Prototypes/Datasets/tips.yml:156]: To add to TsjipTsjip's comment, if "prowlers" is intended to refer to the ship, it becomes a proper noun: consider capitalising the name :)

File: Resources/Prototypes/Datasets/tips.yml [hunk 5]
```diff
@@ -123,3 +123,47 @@
...
+  - "Certain ships can act as motherships. They have shipyard consoles, which allow you to purchase and sell other smaller ships!"
+  - "As a crewmemeber, you can start a mutiny if your captain refuses to pay you. Be careful though, as the captain has the permission to execute the mutiny leader!"
+  - "Your PDA has a pre-installed news application. Check it out, perhaps the local reporter posted some interesting news?"
```
Comment (arimah) [inline Resources/Prototypes/Datasets/tips.yml:159]: Suggested tense change: posted -> has posted

File: Resources/Prototypes/Datasets/tips.yml [hunk 6]
```diff
@@ -123,3 +123,48 @@
...
+  - "Medical ships often have access to the hypospray. An experienced chemist can turn it into a lethal weapon, or use it to safely put people to sleep."
+  - "If you're hurt, try to ask for medical help on the radio. Medics are often eager to heal the wounded, as their job usually gets boring after hours of waiting."
+  - "Always carry a GPS with yourself! It can help you locate your station, or tell your location to others and get saved."
+  - "Smuggling is very profitable, but also very risky! Remember to have an IFF console and hide your IFF before going for a dead drop."
+  - "As a smuggler, try to destroy any evidence of your crimes. Remember, your PDA has a notepad app, which can be used to store important notes in a digital form."
+  - "Security has a special IFF console that completely hides their ship on the radar! If you're an outlaw, watch out for those invisible Prowlers!"
+  - "Certain ships can act as motherships. They have shipyard consoles, which allow you to purchase and sell other smaller ships!"
+  - "As a crewmemeber, you can start a mutiny if your captain refuses to pay you. Be careful though, as the captain has the permission to execute the mutiny leader!"
```
Comment (mnemosynium) [inline Resources/Prototypes/Datasets/tips.yml:158]: Typo in crewmember

File: Resources/Prototypes/Datasets/tips.yml [hunk 7]
```diff
@@ -123,3 +123,48 @@
...
+  - "Your PDA has a pre-installed news application. Check it out, perhaps the local reporter has posted some interesting news?"
+  - "There's a special app installed in your PDA, which allows you to see posted bounties. If you can fulfill one of them, you can ask the local SR or Sheriff for a reward!"
+  - "Remember, Frontier Outpost has a 200m wide safezone. It's forbidden to seriously harm people or property in any way within that limit!"
```
Comment (mnemosynium) [inline Resources/Prototypes/Datasets/tips.yml:161]: Is this strictly true? Presently the rules (and word from admins) only restrict antag/griefing activity within 200m, but not all inter-character conflict full-stop.

File: Resources/Prototypes/Datasets/tips.yml [hunk 8]
```diff
@@ -123,3 +123,48 @@
...
+  - "If you're using an IFF console, remember to turn your IFF on near frontier, or you'll risk facing security on your ship!"
+  - "Going on an expedition alone is very risky! Make sure to always have a crewmate to save you in case you get ambushed by a xeno queen."
+  - "Expeditions can yield rare and useful items, and not all of them are legal. Try not to leave those lying on the floor, as security can board you at any moment!"
```
Comment (mnemosynium) [inline Resources/Prototypes/Datasets/tips.yml:164]: This may misrepresent what security is allowed to do.

Comment (mnemosynium) [reply Resources/Prototypes/Datasets/tips.yml:164]: Changing it to "might" could be sufficient

Comment (Mnemotechnician) [reply Resources/Prototypes/Datasets/tips.yml:164]: Yea, I'm not really good at phrasing things

File: Resources/Prototypes/Datasets/tips.yml (hunk 7 above)
Comment (Mnemotechnician) [reply Resources/Prototypes/Datasets/tips.yml:161]: I'm not sure how to phrase it better, it could help if you gave a better example

Comment (mnemosynium) [reply Resources/Prototypes/Datasets/tips.yml:161]: "Remember, Frontier Outpost has a 200m wide safe-zone. It's forbidden to engage in antagonist behavior or damage property in any way within that limit!"

Something like that?

File: Resources/Prototypes/Datasets/tips.yml [hunk 9]
```diff
@@ -123,3 +123,48 @@
...
+  - "Medical ships often have access to the hypospray. An experienced chemist can turn it into a lethal weapon, or use it to safely put people to sleep."
+  - "If you're hurt, try to ask for medical help on the radio. Medics are often eager to heal the wounded, as their job usually gets boring after hours of waiting."
+  - "Always carry a GPS with yourself! It can help you locate your station, or tell your location to others and get saved."
+  - "Smuggling is very profitable, but also very risky! Remember to have an IFF console and hide your IFF before going for a dead drop."
+  - "As a smuggler, try to destroy any evidence of your crimes. Remember, your PDA has a notepad app, which can be used to store important notes in a digital form."
+  - "Security has a special IFF console that completely hides their ship on the radar! If you're an outlaw, watch out for those invisible Prowlers!"
+  - "Certain ships can act as motherships. They have shipyard consoles, which allow you to purchase and sell other smaller ships!"
+  - "As a crewmember, you can start a mutiny if your captain refuses to pay you. Be careful though, as the captain has the permission to execute the mutiny leader!"
```
Comment (mnemosynium) [inline Resources/Prototypes/Datasets/tips.yml:158]: 1. A lawful mutiny requires the captain to have broken Space Law (non-payment is a civil matter)
2. Non-payment can be resolved through a lawsuit, or security may be willing to intervene
3. An unlawful mutiny would need to first follow escalation guidelines

I think keeping this tip may encourage bad behavior.

Comment (Everturning) [reply Resources/Prototypes/Datasets/tips.yml:158]: i feel like, if you reword it to something along the lines of "Check your space law book, found from your PDA, for some useful information on law! For example: The rules for mutiny on the captain if they don't pay you!

Comment (Everturning) [reply Resources/Prototypes/Datasets/tips.yml:158]: encourages people not only to see IC law, but also achieves some of the same effect

Comment (mnemosynium) [reply Resources/Prototypes/Datasets/tips.yml:158]: +1 for a tip on finding Space Law in your PDA, but mutiny for non-payment is a crime. There is also nothing explicitly worded around non-payment in Space Law.

**Review by FoxxoTrystan — APPROVED** [review]

**Comment by dvir001** [comment]
Giving this a green light since its been few days now.
We should have another one of this in like 2 months if we have stuff to add.
Or before if we need to fix something.
